quasar-kotlin/src/main/resources/META-INF/suspendable-supers

```diff
@@ -0,0 +1,3 @@
+java.util.function.Supplier.*
```

Comment:
The Gradle build expects to _generate_ `META-INF/suspendable-supers` files using the `scanAndIInstrument()` function in `build.gradle`. Adding this `suspendable-supers` file of your own to `src/main/resources` will not "play nicely" with the generated file.

Reply:
Yes this is just a sandpit to try this out. Also as quasar-kotlin tests are loaded and instrumented by the JavaAgent.

The quasar documentation says that suspendable-supers can be added manually. So this is what we've tried in V5. This does seem to then work ok. So I'm not suggesting using the scanner as well. I'd rather just add these manually to suspendable-supers as needed.

Reply:
They _can_ be added manually, _except_ in the Quasar build which expects to generate them using an Ant task.

Reply:
Ok, that is understood, the fixes you have done are just for the quasar build. I've approved them.